src/sonic-host-services/scripts/hostcfgd

```diff
@@ -169,6 +208,29 @@ class AaaCfg(object):
         if modify_conf:
             self.modify_conf_file()
 
+    def tacacs_get_source_intf_ip(self, keys, src_intf):
+        new_ipv4_addr = ""
```

Comment:
Is the goal to get first IPv4 & IPv6 address ?

Can you please change name to something generic like, "ip-addr-parser" ?